bin/nvcc_wrapper

```diff
@@ -90,6 +90,11 @@ replace_pragma_ident=0
 # Mark first host compiler argument
 first_xcompiler_arg=1
 
+# Allow for setting temp dir without setting TMPDIR in parent (see https://docs.olcf.ornl.gov/systems/summit_user_guide.html#setting-tmpdir-causes-jsm-jsrun-errors-job-state-flip-flop)
+if [[ ! -z ${_NVCC_WRAPPER_TMPDIR_OVERRIDE+x} ]]
```

Comment:
What does the `+x` do?

Reply:
+x ensures that if _NVCC_WRAPPER_TMPDIR_OVERRID="", it is still respected.  We can omit this, since one can argue that passing that value makes no sense and shouldn't be respected.  Mostly it's just convention to differentiate between unset and set to empty string.